conda-recipe/run_test.sh

```diff
@@ -15,55 +15,51 @@
 # limitations under the License.
 #===============================================================================
 
-daal4py_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+sklex_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 count=3
 while [[ count -ne 0 ]]; do
-    if [[ -d $daal4py_dir/daal4py/ && -d $daal4py_dir/tests/ && -d $daal4py_dir/examples/daal4py ]]; then
+    if [[ -d $sklex_root/.ci/ && -d $sklex_root/examples/ && -d $sklex_root/tests/ ]]; then
         break
     fi
-    daal4py_dir="$( dirname "${daal4py_dir}" )"
+    sklex_root="$( dirname "${sklex_root}" )"
     count=$(($count - 1))
 done
 
-echo "daal4py_dir=$daal4py_dir"
 if [[ count -eq 0 ]]; then
-    echo "run_test.sh must be in daal4py repository"
+    echo "run_test.sh did not find the required testing directories"
     exit 1
 fi
 
-echo "Start testing ..."
 return_code=0
 
-python -c "import daal4py"
-return_code=$(($return_code + $?))
+if [ -z "${PYTHON}" ]; then
+    export PYTHON=python
+fi
 
-echo "Pytest run of legacy unittest ..."
-echo ${daal4py_dir}
-pytest --verbose -s ${daal4py_dir}/tests
+${PYTHON} -c "from sklearnex import patch_sklearn; patch_sklearn()"
 return_code=$(($return_code + $?))
 
 echo "NO_DIST=$NO_DIST"
 if [[ ! $NO_DIST ]]; then
-    echo "MPI pytest run of legacy unittest ..."
+    echo "MPI pytest run"
```

Comment:
Why would this one have an `echo` entry while the `echo` entries of others were removed?